android/src/reactNativeVersionPatch/nativeHierarchyManager/latest/com/swmansion/reanimated/layoutReanimation/ReanimatedNativeHierarchyManager.java

```diff
@@ -339,6 +346,15 @@ public synchronized void manageChildren(
     super.manageChildren(tag, null, null, tagsToDelete);
   }
 
+  private void maybeNotifyAboutViewsRemoving(int[] tagsToDelete) {
+    if (tagsToDelete != null) {
+      AnimationsManager animationsManager = ((ReaLayoutAnimator) mReaLayoutAnimator).getAnimationsManager();
+      animationsManager.visitTreeForTags(tagsToDelete, false);
```

Comment:
looks like it'd be better to move this logic to AnimationsManager rather than exposing implemnentation details to other classes.